tests/resources/Nimbus-Util.robot

```diff
@@ -451,6 +451,44 @@ Power Off Host
     ${out}=  Execute Command  poweroff -d 0 -f
     Close connection
 
+Static IP Address Create
+    [Timeout]    110 minutes
+    Log To Console  Starting Static IP Address test...
```

Comment:
Update comment to not refer test as this is keyword.

Reply:
Done.